src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests2.cs

```diff
@@ -2285,5 +2287,1769 @@ public static class E
             //         [System.Diagnostics.CodeAnalysis.UnscopedRef]
             Diagnostic(ErrorCode.ERR_UnscopedRefAttributeUnsupportedMemberTarget, "System.Diagnostics.CodeAnalysis.UnscopedRef").WithLocation(8, 10));
     }
+
+    [Fact]
+    public void Cref_01()
+    {
+        var src = """
+/// <see cref="E.extension(int).M(string)"/>
+/// <see cref="E.M(int, string)"/>
+/// <see cref="E.extension(int).M"/>
+/// <see cref="E.M"/>
+static class E
+{
+    extension(int i)
+    {
+        /// <see cref="M(int, string)"/>
+        /// <see cref="M(string)"/>
+        /// <see cref="M"/>
+        public void M(string s) => throw null!;
+    }
+}
+""";
+        var comp = CreateCompilation(src, parseOptions: TestOptions.RegularPreviewWithDocumentationComments);
+        comp.VerifyEmitDiagnostics(
+            // (10,24): warning CS1574: XML comment has cref attribute 'M(string)' that could not be resolved
+            //         /// <see cref="M(string)"/>
```

Comment:
>/// <see cref="M(string)"/> [](http://example.com/codeflow?start=23&length=27)

Should this work instead? Let's capture a design question.

Reply:
Confirmed with Mads over email thread